countermap/countermap.go

```diff
@@ -0,0 +1,170 @@
+package countermap
```

Comment:
I'm not sure where we are using this but, is this not implemented by the Prometheus golang client in `CounterVec`?

Reply:
The `CounterVec` is meant to only increment over time. If using prometheus to track active processes per fingerprint we'd want a `Gauge` type of metric.

The problem with using prometheus is cardinality. We want to to track number of active processes by alert fingerprint, and we could expect an alert fingerprint to be almost always unique per message received. So to use prometheus to track active processes/fingerprint without creating problems running am-executor for long periods of time time we'd also want to take care of deleting metrics after they aren't needed anymore. Instead of spending time looking into that I chose to spend time tracking active processes separately from prometheus.